EoT/embeds.xml

```diff
@@ -0,0 +1,7 @@
+<Ui xmlns="http://www.blizzard.com/wow/ui/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.blizzard.com/wow/ui/ ..\FrameXML\UI.xsd">
```

Comment:
please just include these in the ToC since there's no actual UI

Reply:
Ok this file is now deleted and all dependencies besides LibStub are pulled in from the top level. 

LibStub seems to be a little different, and it doesn't have a .toc file, which I think prevents it from being loaded like the other libraries. I just kept it in the libs folder, and pulled it in through the .toc file.